examples/simple_key_value_store.rs

```diff
@@ -280,7 +278,8 @@ struct Client {
 impl Client {
     fn new(_bootstrap_peers: Vec<Endpoint>) -> Result<Client, RoutingError> {
```

Comment:
Are these bootstrap things not completely take care of in crust ? i.e. via the config file.